rules/o365/credential_access_o365_potential_password_spraying_attack.toml

```
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2020/12/01"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/12/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a high number of failed O365 user authentication attempts from a single IP address, which could be indicative
```

```suggestion
Identifies a high number of failed Microsoft 365 user authentication attempts from a single IP address, which could be indicative
```
Depending on the wording we go with, I think the O365 part should be enumerated out such as Microsoft 365 or Office 365.